### PR TITLE
Issue 574 - Fix drop collection into caroussel tile

### LINKED ISSUE #574: Alpha 12 breaks adding a Collection to a Carousel tile
Steps to reproduce:

1. Add and publish some news items.
2. In Compose view/tab, drag default News collection into a Carousel Tile. It gets green, spinner is briefly displayed but content doesn't get updated. Also noticed there's success (200) for `@@updatetilecontent?tile-type=collective.cover.carousel` when item is dropped.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #574

### CODE REVIEW

File: src/collective/cover/tiles/carousel.py [hunk 1]
```diff
@@ -64,9 +66,14 @@ def populate_with_object(self, obj):
         :param uuids: The list of objects' UUIDs to be used
         :type uuids: List of strings
         """
-        if not self._has_image_field(obj):
-            return
-        super(CarouselTile, self).populate_with_object(obj)
+        # check permission
+        super(ListTile, self).populate_with_object(obj)  # check permission
```
Comment (hvelarde) [inline src/collective/cover/tiles/carousel.py]: why did you replaced `super(CarouselTile, self)`?

**Comment by hvelarde** [comment]
@davilima6 do you mind reviewing this? let me know if it works for you.

**Comment by davilima6** [comment]
@hvelarde, @rodfersou: it does! Thanks! That's a very important one.

**Comment by hvelarde** [comment]
@davilima6 I haven't finished the code review :-)

**Pull request merged**